mne/beamformer/tests/test_lcmv.py

```diff
@@ -689,6 +689,31 @@ def test_lcmv_reg_proj(proj):
                         weight_norm='nai', rank=None, verbose=True)
     want_rank = 302  # 305 good channels - 3 MEG projs
     assert filters['rank'] == want_rank
+    # And also with and without noise_cov
+    with pytest.raises(ValueError, match='several sensor types'):
```

Comment:
@britta-wstnr I added these lines (which ensure that `cov=None` gives the same STCs as `cov=make_ad_hoc_cov(...)`) and they passed on `master`, and they still pass here.

Internally the `filters['weights']` get scaled, but they get scaled by the inverse of the scaling in the `whitening` matrix, so it should be fine.

The only difference is that the `allclose(np.dot(...))` test below on `master` had to be the following (since `whitener` there is `None`):
```
    assert_allclose(
        filters_nocov['weights'],
        np.dot(filters_adhoc['weights'], filters_adhoc['whitener']))
```